src/training_strategies.jl

```diff
@@ -296,3 +296,80 @@ function get_loss_function(loss_function, lb, ub, eltypeθ, strategy::Quadrature
     loss = (θ) -> 1 / area * f_(lb, ub, loss_function, θ)
     return loss
 end
+
+struct WeightedGridTraining <: AbstractTrainingStrategy
```

Comment:
Missing documentation

Reply:
Will work on this. I can also add the weights and samples field directly into GridTraining if that works